inc/field/ldapselectfield.class.php

```diff
@@ -120,7 +120,7 @@ public function getAvailableValues() {
          return [];
       }
 
-      $ldap_values   = json_decode(plugin_formcreator_decode($this->question->fields['values']));
```

Comment:
plugin_formcreator_encode() is not used in method PrepareQuestionInputForSave() then we should be able to remove plugin_formcreator_decode()